packages/calypso-e2e/src/lib/pages/index.ts

```diff
@@ -1,34 +1,35 @@
-export * from './published';
-export * from './login-page';
+export * from './account-closed-page';
+export * from './cart-checkout-page';
+export * from './checkout-thank-you-page';
+export * from './domains-page';
 export * from './editor-page';
-export * from './my-home-page';
-export * from './marketing-page';
+export * from './full-site-editor-page';
 export * from './general-settings-page';
-export * from './themes-page';
-export * from './themes-detail-page';
-export * from './media-page';
-export * from './plans-page';
 export * from './individual-purchase-page';
-export * from './cart-checkout-page';
-export * from './stats-page';
-export * from './site-import-page';
 export * from './invite-people-page';
+export * from './login-page';
+export * from './marketing-page';
+export * from './media-page';
+export * from './my-home-page';
+export * from './p2-page';
+export * from './pages-page';
 export * from './people-page';
+export * from './plans-page';
+export * from './plugins-page';
+export * from './posts-page';
+export * from './published';
 export * from './reader-page';
+export * from './revisions-page';
+export * from './shared-types';
 export * from './signup/user-signup-page';
-export * from './account-closed-page';
-export * from './domains-page';
+export * from './site-import-page';
+export * from './stats-page';
+export * from './themes-detail-page';
+export * from './themes-page';
 export * from './use-a-domain-i-own-page';
-export * from './p2-page';
 export * from './woocommerce-landing-page';
-export * from './posts-page';
-export * from './pages-page';
-export * from './plugins-page';
 export * from './writing-settings-page';
-export * from './shared-types';
-export * from './full-site-editor-page';
-export * from './checkout-thank-you-page';
 
-export * from './me';
 export * from './external';
+export * from './me';
 export * from './signup';
```

Comment:
No changes here, just sorted it alphabetically! 😅

Reply:
Much needed --- thank you 😁